### PR TITLE
docs(queues): added JOB_REF provider

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Docs
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
This PR, added JOB_REF provider

### CODE REVIEW

**Comment by Tony133** [comment]
reference issue: https://github.com/nestjs/docs.nestjs.com/issues/1557

to be evaluated

File: content/techniques/queues.md [hunk 1]
```diff
@@ -194,6 +194,25 @@ const job = await this.audioQueue.add(
...
+Whether the processor needs to depend on a request scoped provider or be explicitly registered as a scoped processor, as follows:
+
+```typescript
+@Processor({
+  name: 'audio',
+  scope: Scope.REQUEST,
+})
+```
+
+and then you can inject the JOB_REF via the constructor.
```
Comment (jmcdo29) [inline content/techniques/queues.md:208]:
```suggestion
then you can inject the `JOB_REF` via the constructor.
```

File: content/techniques/queues.md [hunk 2]
```diff
@@ -194,6 +194,25 @@ const job = await this.audioQueue.add(
...
+
+```typescript
+constructor(@Inject(JOB_REF) jobRef: Job) {
+   console.log(jobRef);
+}
+```
+
```
Comment (jmcdo29) [inline content/techniques/queues.md:215]: I assume `JOB_REF` can be imported from `@nestjs/bull`, can we add the hint for that?

**Review by jmcdo29 — CHANGES_REQUESTED** [review]

File: content/techniques/queues.md (hunk 2 above)
Comment (Tony133) [reply content/techniques/queues.md:215]: is there also at the beginning of the article you say to repeat it?

Comment (jmcdo29) [reply content/techniques/queues.md:215]: I see information about jobs, but nothing about the `JOB_REF` constant. Can you point that out to me?

Comment (Tony133) [reply content/techniques/queues.md:215]: I got confused :-), such a thing could also:

``` import { JOB_REF } from '@nestjs/bull' ```

Comment (jmcdo29) [reply content/techniques/queues.md:215]: That should be fine.

**Comment by Tony133** [comment]
wrong pull

**Pull request closed without merging**